storage/src/system/auction/detail.rs

```diff
@@ -135,6 +135,7 @@ impl ValidatorBidsDetail {
                     vesting_schedule_period_millis,
                 )
                 && !v.inactive()
+                && !v.staked_amount() >= U512::one()
```

Comment:
Is this meant to be one here, or should it be the min_bid_amount?

Reply:
I guess the `inactive` latch keeps us safe, but maybe we should also have this be the `minimum_bid_amount` instead of one?

Reply:
its meant to be 1 here
the min bid enforcement happens in the winners selection

Reply:
Noted